s3prl/upstream/byol_a/expert.py

```diff
@@ -11,9 +11,9 @@
 
```

Comment:
https://github.com/s3prl/s3prl/blob/4f5ad57dbff1b10234b0cb1cc35f03b64719a60a/s3prl/upstream/byol_a/expert.py#L2-L7

Feel free to add your name and the institution as the author.

Reply:
Thanks, but I think I should refrain from touching on this.